itwm_example/mco/subprocess_workflow_evaluator.py

```diff
@@ -36,7 +36,7 @@ def _call_subprocess(self, command, user_input):
             env=env,
```

Comment:
We can remove this file, and any unit testing code that goes with it since we took out the `MCOCommunicator`

Reply:
We will create a new PR for this, as discussed offline. Thank you for pointing it out.